src/modules/rewards/services/arb-rebate-service.ts

```diff
@@ -268,18 +267,25 @@ export class ArbRebateService {
       };
     }
 
+    const inputTokenPrice = deposit.price.usd;
     const historicRewardTokenPrice = await this.marketPriceService.getCachedHistoricMarketPrice(
       DateTime.fromJSDate(deposit.depositDate).minus({ days: 1 }).toJSDate(),
       rewardToken.symbol.toLowerCase(),
     );
+
+    const inputAmountUsd = new BigNumber(deposit.amount)
+      .multipliedBy(inputTokenPrice)
+      .dividedBy(new BigNumber(10).pow(deposit.token.decimals));
+    const maxRewardsUsd = inputAmountUsd.multipliedBy(REWARDS_PERCENTAGE_LIMIT);
     const rewardsUsd = new BigNumber(bridgeFeeUsd).multipliedBy(ARB_REBATE_RATE).toFixed();
+    const cappedRewardsUsd = BigNumber.min(maxRewardsUsd, rewardsUsd).toFixed();
```

Comment:
Does `.toFixed()` in this case default to some decimal precision?

Reply:
No, it doesn't

Reply:
What is the precision of using `toFixed` without any params?

Reply:
It returns the string representation of the number without rounding it. [This](https://mikemcl.github.io/bignumber.js/#toFix) are the docs for this method